scalafix/input/src/main/scala/synthetic/Example2.scala

```diff
@@ -0,0 +1,21 @@
+/*
+rule = MigrationRule
+*/
+package synthetic
+
+class Example2 {
```

Comment:
Maybe we should add all the incompat files in the scalafix tests

Reply:
ok